content/en/docs/Overview/your-first-pipeline.md

```diff
@@ -19,106 +19,106 @@ You can start a pipeline manually, or you can configure it to be automatically t
 - A security group within AWS with appropriate permissions
 - A Load Balancer
```

Comment:
How about linking to the Load Balancers page?

Reply:
For this section, how about replacing Prerequisites with 
```
## {{%heading "prereq" %>}}
```

Reply:
This requires a little extra copy, which I'm fine with adding.